build.sbt

```diff
@@ -11,13 +11,13 @@ import scala.xml.transform.{RewriteRule, RuleTransformer}
 // Global settings
 organization in ThisBuild := "org.http4s"
 version      in ThisBuild := scalazCrossBuild("0.15.0-SNAPSHOT", scalazVersion.value)
-apiVersion   in ThisBuild <<= version.map(extractApiVersion)
+apiVersion   in ThisBuild := version.map(extractApiVersion).value
```

Comment:
Yes.  I think this could also be expressed as `extractApiVersion(version.value)`.